README.md

```diff
@@ -57,9 +57,21 @@ You can also configure the path to the settings file with environment variable `
 
 The http port can be configured separately with environment variable `PORT`. You can also [run on port 80 with systemd](https://github.com/tkurki/marinepi-provisioning/blob/d3d624629799a3b96234a90fc42bc22dae4fd3a2/roles/node-app/templates/node_app_systemd_socket.j2). Environment variable NMEA0183PORT sets the NMEA 0183 tcp port.
 
+Storing Configuration Outside The Server Install Directory
+==========================================================
+You can store configuration like the settings file, plugin cofiguration, defaults.js and the mapcache in a directory outside of the server install using the `-c` option (or the `SIGNALK_NODE_CONDFIG_DIR` env variable).
+
+For example, `./bin/signalk-server -c /usr/local/etc/node_server_config -s settings/my-boat.json`
```

Comment:
I don't think the `settings` directory makes any sense here, as it exists to group the different examples together in one directory, and in this case there will be just one that could have a standard name, foregoing the need for the `-s` argument.